components/resource-creation/ResourceDisplay.tsx

```diff
@@ -96,7 +96,7 @@ function ResourceDisplay() {
           measureBundle.content,
           measurementPeriod.start?.toISOString(),
           measurementPeriod.end?.toISOString(),
-          useTrustMetaProfile
+          trustMetaProfile
```

Comment:
I don't think this argument goes anywhere. It's not in the signature of `calculateDetailedResult`.

Reply:
Never mind! Github was showing me an out of date copy of `calculateDetailedResult`.